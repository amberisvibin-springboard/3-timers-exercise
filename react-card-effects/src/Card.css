/* .Card {
    height: 314px;
    width: 226px;
} */