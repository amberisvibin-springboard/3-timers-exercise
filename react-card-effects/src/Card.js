import "./Card.css";

function Card(props) {
  return (
    <div className="Card">
      <img src={props.img} height="314px" width="226px"></img>
    </div>
  );
}
export default Card;
