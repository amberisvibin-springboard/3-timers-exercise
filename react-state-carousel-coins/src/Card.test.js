import { render } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

//smoke test
it("renders without crashing", function () {
  render(
    <Card caption={"test"} src={TEST_IMAGES[0]} currNum={0} totalNum={1} />
  );
});

//snapshot test
it("matches snapshot", function () {
  const { asFragment } = render(
    <Card caption={"test"} src={TEST_IMAGES[0]} currNum={0} totalNum={1} />
  );
  expect(asFragment()).toMatchSnapshot();
});
