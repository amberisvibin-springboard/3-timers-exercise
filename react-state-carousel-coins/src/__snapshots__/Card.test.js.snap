// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="[object Object]"
    />
    <small
      class="Card-small"
    >
      Image 0 of 1.
    </small>
  </div>
</DocumentFragment>
`;
